frontend/src/pages/detail/BBDetail.tsx

```diff
@@ -406,26 +465,57 @@ export default function BBDetail() {
     });
   }, []);
 
+  // The navigator to switch pages
+  const navigate = useNavigate();
+
   return (
     <div>
       <Toast ref={toast} />
       <div className="grid flex justify-content-center">
-        <link
-          rel="stylesheet"
-          href="https://unpkg.com/primeflex@3.1.2/primeflex.css"
-        ></link>
-        <div className="col-11">
-          <div className="pt-2">
+        <div className="flex col-12 p-0">
+          <div className="flex col-1">
+            <Button
+              type="button"
+              label="Back"
+              icon="pi pi-arrow-left"
+              onClick={() => navigate("/buy-backs")}
+              className="p-button-sm my-auto ml-1"
+            />
+          </div>
+          <div className="pt-2 col-10">
             {isBBAddPage ? (
               <h1 className="p-component p-text-secondary text-5xl text-center text-900 color: var(--surface-800);">
-                Add Buy Back Sales
+                Add Buy Back Sale
```

Comment:
Same here, I think this should be "Add Book Buyback", and "Modify Book Buyback" right below